### ISSUE TITLE
Error while installing rmt_agent

### ISSUE THREAD

**Issue by evshary** [Adlink-ROS/RMT_example#25]
![image](https://user-images.githubusercontent.com/456210/122880937-29bbc680-d36d-11eb-80a9-1bf607ab7e0b.png)

**Comment by evshary** [comment]
This is fixed via #26 
But I think we can agent_example should not run as daemon, since systemd can run it directly.

**Comment by evshary** [comment]
We can control rmt-agent with

```bash
systemctl --user stop rmt-agent
systemctl --user start rmt-agent
```
